src/Compilers/Core/Portable/Symbols/SymbolEqualityComparer.cs

```diff
@@ -23,6 +23,7 @@ public sealed class SymbolEqualityComparer : IEqualityComparer<ISymbol>
 
         // Internal only comparisons:
         internal readonly static SymbolEqualityComparer ConsiderEverything = new SymbolEqualityComparer(TypeCompareKind.ConsiderEverything);
+        internal readonly static SymbolEqualityComparer IgnoreNullableModifiersForReferenceTypes = new SymbolEqualityComparer(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes);
```

Comment:
>internal readonly static SymbolEqualityComparer IgnoreNullableModifiersForReferenceTypes = new SymbolEqualityComparer(TypeCompareKind.IgnoreNullableModifiersForReferenceTypes); [](start = 8, length = 176)

We already have ```TypeSymbol.EqualsIgnoringNullableComparer```. Is it not suitable?

Reply:
Thanks. I looked around but must have missed it.

---
In reply to: [321431891](https://github.com/dotnet/roslyn/pull/38460#discussion_r321431891) [](ancestors = 321431891)